.github/workflows/stackhpc-all-in-one.yml

```
@@ -404,6 +404,17 @@ jobs:
         if: inputs.upgrade
 
+      - name: Install kernel-modules-extra package before updating DNF repositories
+        run: |
+          docker run -t --rm \
+            -v $(pwd):/stack/kayobe-automation-env/src/kayobe-config \
+            -e KAYOBE_ENVIRONMENT -e KAYOBE_VAULT_PASSWORD -e KAYOBE_AUTOMATION_SSH_PRIVATE_KEY \
+            $KAYOBE_IMAGE \
+            /stack/kayobe-automation-env/src/kayobe-config/.automation/pipeline/overcloud-host-command-run.sh --become --command "dnf install -y kernel-modules-extra"
+        env:
+          KAYOBE_AUTOMATION_SSH_PRIVATE_KEY: ${{ steps.ssh_key.outputs.ssh_key }}
+        if: inputs.upgrade and inputs.os_distribution == 'rocky'
```

actionlint (1.7.12)

[error] parser did not reach end of input after parsing the expression. 6 remaining token(s) in the input: "IDENT", "IDENT", ".", "IDENT", "==", "STRING"

(expression)